src/jit/codegencommon.cpp

```diff
@@ -11276,6 +11276,25 @@ bool Compiler::IsRegisterPassable(GenTreePtr tree)
 {
     return IsRegisterPassable(gtGetStructHandleIfPresent(tree));
 }
+
+// Returns true if the type is returned in multiple registers
+// 
+// Arguments
+//     hClass   -  type handle
+//
+// Returns
+//     true if type is returned in multiple registers
+bool Compiler::IsMultiRegRetVal(CORINFO_CLASS_HANDLE hClass)
+{
+    if (hClass == NO_CLASS_HANDLE)
+    {
+        return false;
+    }
+
+    SYSTEMV_AMD64_CORINFO_STRUCT_REG_PASSING_DESCRIPTOR structDesc;
+    eeGetSystemVAmd64PassStructInRegisterDescriptor(hClass, &structDesc);
+    return structDesc.passedInRegisters && structDesc.eightByteCount == CLR_SYSTEMV_MAX_EIGHTBYTES_COUNT_TO_PASS_IN_REGISTERS;
```

Comment:
shouldn't the expression be "structDesc.eightByteCount > 1"?

---
In reply to: [58158201](https://github.com/dotnet/coreclr/pull/4042#discussion_r58158201) [](ancestors = 58158201)